tests/sentry/integrations/slack/webhooks/actions/test_status.py

```diff
@@ -76,7 +76,7 @@ def get_original_message_block_kit(self, group_id):
 
     def get_ignore_status_action(self, text, selection):
         return {
-            "action_id": "status",
```

Comment:
since the tests here had the original name for ignore buttons hardcoded here (function used only for block kit tests), they still passed; changed it to be the same as value to verify that the action is handled with the new names.